src/auth/token-generator.ts

```diff
@@ -254,13 +255,38 @@ export class FirebaseTokenGenerator {
    *                           service account key and containing the provided payload.
    */
   public createCustomToken(uid: string, developerClaims?: {[key: string]: any}): Promise<string> {
+    return this.createCustomTokenInternal(uid, null, developerClaims);
+  }
+
+  /**
+   * Creates a new Firebase Auth Custom token.
+   *
+   * @param {string} uid The user ID to use for the generated Firebase Auth Custom token.
```

Comment:
I know this is the current pattern but we should move away from re-stating the types, eg. `string` in the javadocs.
This is according to the internal typescript readability guideline.